index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Quiz 1</title>
+    <link rel="stylesheet" type="text/css" href="main.css">
+</head>
+<body>
+        <article class="cards1">
+        <header>
+            <h1>German Chocolate Cake</h1>
+        </header>
+        <section>
+        <element>
+<img class="byitself" src="http://www.keeponcaking.com/wp-content/uploads/2010/10/finished-cake.jpg" width="400" height="300">
+        </element>
+        <element class="pad">
+            Dark chocolate cake covered in rich German Chocolate icing, topped with strawberries.
+        </element>
+        <element class="special">
+            Available
+        </element>
+        </section>
+        <section class="space", "volume">
+            <h4>Specifications</h4>
+            <element>Size: 12" Round<br>Weight: 24 Ounces</element>
```

Comment:
In the requirements this was specified to be two block level elements, not just one element with a `<br>` tag in the middle.